aiogram/types/fields.py

```diff
@@ -169,7 +169,7 @@ class DateTimeField(Field):
     """
 
     def serialize(self, value: datetime.datetime):
-        return round(value.timestamp())
+        return (value - datetime.datetime(1970, 1, 1)).total_seconds()
```

Comment:
round returns int, total_seconds() returns float, so this methods in not equivalent
Also this dirty-hack is needed only on Windows, so please make an platform-specific function and use it here.

For example at module root (pseudo-code):
```python
if sys.platform == "win32":
    def _serialize_datetime(value):
        return round(...total_seconds())
else:
    def _serialize_datetime(value):
        return round(...timestamp())
```